float64                  expected_speed
ball_filter/Plane     fitted_plane
ball_filter/LKFState  lkf_state

